test/inductor/test_torchinductor_opinfo.py

```diff
@@ -356,6 +356,8 @@ def format_op(op):
     "nn.functional.multilabel_margin_loss": {f16},
     "nn.functional.multi_margin_loss": {f16},
     "nn.functional.avg_pool3d": {f16},
+    # not implemented for 'Bool'
+    "nn.functional.unfold": {b8},
```

Comment:
https://github.com/pytorch/pytorch/pull/137873 will cover this change.

Reply:
Just for CI, I will resolve this conflict after #137873 is merged.